ethereum/eth/src/main/java/tech/pegasys/pantheon/ethereum/eth/sync/BlockPropagationManager.java

```diff
@@ -105,9 +110,34 @@ private void setupListeners() {
         .subscribe(EthPV62.NEW_BLOCK_HASHES, this::handleNewBlockHashesFromNetwork);
   }
 
+  @VisibleForTesting
+  void broadcastBlock(final Block block) {
+    blockBroadcaster = new BlockBroadcaster(ethContext);
```

Comment:
Don't create a new `BlockBroadcaster` here, just use the one passed into the constructor.